iscservice/isc.c

```diff
@@ -94,9 +94,16 @@ struct m0_sm_state_descr isc_fom_phases[] = {
 
 static size_t fom_home_locality(const struct m0_fom *fom)
 {
-	struct m0_isc_comp_req *comp_req = M0_AMB(comp_req, fom, icr_fom);
+	struct m0_fop_isc *fi = m0_fop_data(fom->fo_fop);
+	size_t             hash;
+
+	M0_ENTRY("fom=%p cob="FID_F, fom, FID_P(&fi->fi_cob));
+
+	hash = m0_fid_hash(&fi->fi_cob);
+
+	M0_LEAVE("hash=%zx", hash);
 
-	return m0_fid_hash(&comp_req->icr_comp_fid);
+	return m0_fid_hash(&fi->fi_cob);
```

Comment:
"return hash;" from here instead of additional m0_fid_hash().

Reply:
@madhavemuri, sorry - I'm not sure I understand that. Which hash?

Reply:
The 'hash‘ is already computed and stored in this variable.
So, as @madhavemuri said, we can have:
```
        return hash;
```

Reply:
Ah, sorry - I just missed the context. Yes, you are right. Will fix it now.